src/fides/api/models/property.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from sqlalchemy import Column, String
+from sqlalchemy.ext.declarative import declared_attr
+
+from fides.api.db.base_class import Base
+from fides.api.db.util import EnumColumn
+from fides.api.schemas.property import PropertyType
+from fides.config import get_config
+
+CONFIG = get_config()
+
+
+class Property(Base):
```

Comment:
given how generic and overloaded a term Property is (i know, not your choice!), and also how generic the fields/columns are for this table, i'd definitely include a docstring here to give more context on what we mean by Property in this class/table!

i'll let you come up with the wording for that 😅 but happy to give a review or brainstorm with ya if you're stuck